### ISSUE TITLE
Errror while creating new project and study that extends 'All TCGA Data'

### ISSUE THREAD

**Issue by kleisb** [BlinkUX/ISB-CGC#171]
Test data is attached.

[Bindea_immune_scores_20151104.16_char_barcodes.tsv.zip](https://github.com/BlinkUX/ISB-LSDF/files/147537/Bindea_immune_scores_20151104.16_char_barcodes.tsv.zip)

To replicate:
- Create new project and study
- Check `Extends an existing project's study data`
- Select `All TCGA Data`
- Configure all fields of the file schema to decimal, except sample barcode field

Stack trace from application log:

```
ERROR    2016-02-26 01:11:35,931 base.py:231] Internal Server Error: /projects/data/
Traceback (most recent call last):
  File "/home/vagrant/www/lib/django/core/handlers/base.py", line 111, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/vagrant/www/lib/django/contrib/auth/decorators.py", line 22, in _wrapped_view
    return view_func(request, *args, **kwargs)
  File "/home/vagrant/www/projects/views.py", line 167, in upload_files
    study.save()
  File "/home/vagrant/www/lib/django/db/models/base.py", line 591, in save
    force_update=force_update, update_fields=update_fields)
  File "/home/vagrant/www/lib/django/db/models/base.py", line 619, in save_base
    updated = self._save_table(raw, cls, force_insert, force_update, using, update_fields)
  File "/home/vagrant/www/lib/django/db/models/base.py", line 681, in _save_table
    forced_update)
  File "/home/vagrant/www/lib/django/db/models/base.py", line 725, in _do_update
    return filtered._update(values) > 0
  File "/home/vagrant/www/lib/django/db/models/query.py", line 600, in _update
    return query.get_compiler(self.db).execute_sql(CURSOR)
  File "/home/vagrant/www/lib/django/db/models/sql/compiler.py", line 1004, in execute_sql
    cursor = super(SQLUpdateCompiler, self).execute_sql(result_type)
  File "/home/vagrant/www/lib/django/db/models/sql/compiler.py", line 775, in execute_sql
    sql, params = self.as_sql()
  File "/home/vagrant/www/lib/django/db/models/sql/compiler.py", line 969, in as_sql
    val = field.get_db_prep_save(val, connection=self.connection)
  File "/home/vagrant/www/lib/django/db/models/fields/related.py", line 1735, in get_db_prep_save
    return self.related_field.get_db_prep_save(value, connection=connection)
  File "/home/vagrant/www/lib/django/db/models/fields/__init__.py", line 627, in get_db_prep_save
    prepared=False)
  File "/home/vagrant/www/lib/django/db/models/fields/__init__.py", line 907, in get_db_prep_value
    value = self.get_prep_value(value)
  File "/home/vagrant/www/lib/django/db/models/fields/__init__.py", line 915, in get_prep_value
    return int(value)
ValueError: invalid literal for int() with base 10: 'All TCGA Data'
```

**Comment by phyllers** [comment]
The underlying issue is the fact that the All TCGA Data project doesn't exist.

**Comment by phyllers** [comment]
removing option until we fix #115 

https://github.com/isb-cgc/ISB-CGC-Webapp/commit/39f8ee92c773869702575a75253ca4abe2763c63
